src/main/java/com/recap/processor/ReCapXmlRouteBuilderPublisher.java

```diff
@@ -69,10 +76,58 @@ public void process(Exchange exchange) throws Exception {
       }
     }).handled(true);
 
-    from("file:" + scsbexportstaging + "?maxMessagesPerPoll=1")
-        .split(body().tokenizeXML("bibRecord", "")).streaming()
+    if (EnvironmentConfig.ONLY_DO_UPDATES) {
+      ZipFileDataFormat zipFile = new ZipFileDataFormat();
+      zipFile.setUsingIterator(true);
+      
```

Comment:
In the two `from`s that follow, you're calling `from` on two sftp paths that are identical except for the directory. Maybe in java it's better to not create unnecessary variables or utility functions, but this could be easier to debug if they were built in the same place - for example with a `buildSftpUrl(String directory)` utility function? Just a suggestion.